### DISCUSSION TITLE
Localization Updates

### DISCUSSION THREAD

**Discussion by mochi-co** [mochi-mqtt/server#311] [General]
This discussion is used to track and notify translators about changes made to translatable content (e.g. README.md). Use @mochi-mqtt/translators to notify localization team.

**Comment by mochi-co** [comment]
**README.md** has been updated:
* 858a28ea4b7e0ff12eeab2cd20c587cb4c706f1f
* 82f32cf53d548a8567d7c641dbf4de01afc6c6ce

**Reply by mochi-co** [reply]
cc @werbenhu

**Reply by werbenhu** [reply]
README-CN.md has been updated to v2.4.1.
